apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -7,20 +18,46 @@ export interface MultilingualTextPresenterProps {
 export const MultilingualTextPresenter = ({
     text,
 }: MultilingualTextPresenterProps): JSX.Element => {
+    const { defaultLanguageCode } = useContext(ConfigurableContentContext);
+
     const { items } = text;
 
-    if (!Array.isArray(items)) {
+    const filteredItems = items.filter((item) => item.languageCode === defaultLanguageCode);
```

Comment:
This is the right idea. However, it's not robust to the case that there is no item with the `defaultLanguageCode` (and this does occur in the real data). Try this instead:
```ts
const itemInDefaultLanguage = items.find(
  ({languageCode}) => languageCode === defaultLanguageCode
)
```
then below
```tsx
{
     isNullOrUndefined(itemInDefaultLanguage) ? null : 
     (
            <PrimaryLanguageTextPresenter text={itemInDefaultLanguage} />
)
}
```
where the `PrimaryLanguageTextPresenter` is up to you to define. It should
- have more prominent text than the presenter you use for the translation items
- occur before the translation items